### PR TITLE
add changes to prevent the application crash on andriod and IOS

### PR DESCRIPTION
**Pull request by sunchunqiang**
This sample application always crash on both Android and IOS platform. Added this change to prevent from the crashing issue.

### CODE REVIEW

File: XamarinSamples/DynamoDB/ContactManager.iOS/Info.plist [hunk 1]
```diff
@@ -51,5 +53,25 @@
 	</array>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+  <key>FacebookAppID</key>
+  <string>2094944187494039</string>
```
Comment (rubyrose-alexis) [inline XamarinSamples/DynamoDB/ContactManager.iOS/Info.plist]: Change this to xxx

File: XamarinSamples/DynamoDB/ContactManager.iOS/Info.plist [hunk 2]
```diff
@@ -21,11 +21,13 @@
...
 	<key>MinimumOSVersion</key>
-	<string>6.0</string>
+  <!--Buding of the project will fail with error [Compiling IB documents for earlier than iOS 7 is no longer supported.] in visual studio 2017.
+  To resolve this build error, we can change this vaule from 6.0 to 7.0-->
+  <string>7.0</string>
 	<key>CFBundleDisplayName</key>
 	<string>My Contacts</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.amazonaws.unity</string>
+	<string>com.amazonaws.xamarin</string>
```
Comment (rubyrose-alexis) [inline XamarinSamples/DynamoDB/ContactManager.iOS/Info.plist]: consider changing this

File: XamarinSamples/DynamoDB/readme.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# AWS SDK for Dot Net - Xamarin DynamoDB Sample
+
+  
+
+##### Configure the DynamoDB Sample
+
+#####Issues may be encountered
```
Comment (rubyrose-alexis) [inline XamarinSamples/DynamoDB/readme.md]: Add a space between # and Issues: "#####Issues may be encountered"  >>> "##### Issues may be encountered"

Comment (sunchunqiang) [reply XamarinSamples/DynamoDB/readme.md]: Changes according to the comments are done.

**Review by rubyrose-alexis — APPROVED** [review]

**Pull request merged**